helm_deploy/values-preprod.yaml

```diff
@@ -13,4 +13,5 @@ env:
   SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json
   OAUTH_ROOT_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth/
   COMMUNITY_ENDPOINT_URL: https://community-api-secure.pre-prod.delius.probation.hmpps.dsd.io/secure
-  HMPPS_TIER_ENDPOINT_URL: https://hmpps-tier-preprod.hmpps.service.justice.gov.uk
\ No newline at end of file
+  HMPPS_TIER_ENDPOINT_URL: https://hmpps-tier-preprod.hmpps.service.justice.gov.uk
+  FLAGS_ENABLEDELIUSTIERUPDATES: false
```

Comment:
It's enabled in preprod at the moment - I think we should leave it on so we can check that everything works as soon as possible. Also I want people outside this team to get used to the fact that writeback is on by default